extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -93,6 +93,7 @@ class AmpSocialShare extends PreactBaseElement {
     return dict({
       'width': responsive || this.element.getAttribute('width'),
       'height': responsive || this.element.getAttribute('height'),
+      'customIcon': this.element.hasAttribute('custom-icon'),
```

Comment:
This can be specified in `AmpSocialShare['props']`

Reply:
Thanks, updated!